server-adapters/resteasy-undertow-cdi/src/main/java/dev/resteasy/embedded/server/UndertowBuilderConfigurer.java

```diff
@@ -0,0 +1,11 @@
+package dev.resteasy.embedded.server;
+
+import io.undertow.Undertow;
+
+/**
+ * This interface is discovered as a ServiceLoader for configuring the {@link Undertow.Builder}
+ * before it builds the {@link Undertow} and starts it.
+ */
+public interface UndertowBuilderConfigurer {
+    void configureUndertowBuilder(Undertow.Builder builder);
```

Comment:
We could make this a generic `configure(Undertow.Builder builder)` method name.

Reply:
Done